lib/galaxy/tools/data_manager/manager.py

```diff
@@ -11,7 +11,10 @@
     send_control_task
 )
 from galaxy.tools.data import TabularToolDataTable
-from galaxy.tools.toolbox.watcher import get_tool_conf_watcher
+from galaxy.tools.toolbox.watcher import (
+    get_loc_dir_watcher,
```

Comment:
I feel that using `tool_data_dir` instead of `loc_dir` everywhere would be better for consistency, if I'm not missing something.